src/makeChange.js

```diff
@@ -1,3 +1,31 @@
-export default function makeChange({price, amountGiven}) {
-  // your code here
+export default function makeChange(price, amountGiven) {
+  let change = {
+    quarters: 0,
+    dimes: 0,
+    nickels: 0,
+    pennies: 0,
+  }
+
+  if(price === amountGiven) {
+    return change
+  }
+
+  let diff = amountGiven - price
+
+  while(diff != 0) {
+    if(diff >= 25) {
+      diff -= 25
+      change.quarters += 1
+    } else if(diff >= 10) {
+      diff -= 10
+      change.dimes += 1
+    } else if(diff >= 5) {
+      diff -= 5
+      change.nickels += 1
+    } else if(diff >= 1){
```

Comment:
Haha I've been pretty repetitive with the use of if else statements. Maybe switch may have been a better solution XD